index.bs

```diff
@@ -302,6 +302,11 @@ one line except when a line break is definitely necessary.</p>
 target WebVTT cues and regions in the video using the ''::cue'', ''::cue()'', ''::cue-region'' and
 ''::cue-region()'' pseudo-elements.</p>
 
+<p>Only a limited set of CSS styles is allowed in WebVTT because user agents that do not support CSS
+will need to implement CSS functionality equivalents. Browsers therefore also have to be limited in
+what CSS styles they support for WebVTT so as to enable identical rendering in browsers and
+non-browsers.</p>
```

Comment:
I am not clear that this is a genuine requirement, that rendering is identical in browsers and non-browsers. If I recall correctly WebVTT has no requirements document. Pointer to a WG resolution that this is important?

Reply:
Actually this sentence is subtly wrong; user agents that don't support CSS (a conformance class as Andreas notes), well, do not support CSS.  UAs that DO support CSS *can* use a full CSS engine but can also implement this restricted subset of CSS.

Reply:
OK, I'll reformulate.